capstone/src/main/webapp/script.js

```diff
@@ -118,6 +118,9 @@ function createMap() {
 }
 
 function getSearchResults() {
+  document.getElementById("map").style.width = "75%"; 
```

Comment:
OOC why is width and style and whatnot set in the script? I would expect it in the CSS.

Reply:
I think this is changing the map view from 100% -> 75%, so 25% of screen width is used for panel width

Reply:
Thanks Grace, LGTM.